### PR TITLE
Update UserGuide

### LINKED ISSUE #58: User guide command and legend inconsistency
image

### PR DESCRIPTION
**Pull request by fullfatwasabi**
closes #58

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -70,50 +78,22 @@ View the staff details of a single staff.
...
 
-Adds a staff to the stored information. The tags and information are optional and can be presented in any order.
-Upon creation of a staff, the system creates a staff ID for them which can be used to refer to them and access the system.
+Adds a staff to the stored information. Information can be presented in any order.
+Upon creation of a staff, the system creates a staff `index` for them which can be used to refer to them and access the system.
 
 Format:
 
-`add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/Tag`
+`add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [r/ROLE] [s/STATUS]`
```
Comment (Gabau) [inline docs/UserGuide.md:90]: Aren't the square brackets inconsistent?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -309,15 +279,15 @@ _Details coming soon ..._
 
 Action | Format, Examples
 --------|------------------
-**View** | `view n/name` <br> `view id/ID`
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/Tag`
-**Delete** | `delete n/name` <br> `delete id/ID` <br> `delete t/group`
-**Edit** | `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]` <br> `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]` <br> `edit ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]`
+**View** | `view n/name` <br> `view i/index`
+**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [r/ROLE] [s/STATUS]`
```
Comment (Gabau) [inline docs/UserGuide.md:283]: Same for these.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -218,66 +194,60 @@ Adds a time period where the staff is working to the staff’s schedule.
...
-`deleteSchedule [n/name] [t/nonrecurr]  [d/date] [startTime-endTime]` \
-`deleteSchedule [id/ID] [t/nonrecurr] [d/date] [startTime-endTime]`
+`deleteSchedule n/name d/fullDayName-shiftNumber` \
+`deleteSchedule i/index d/fullDayName-shiftNumber`
 
 Examples:
 
-`deleteSchedule n/Joe Mon-0800-1200` \
-`deleteSchedule id/12345678 t/nonrecurr d/2021-09-09 0800-1200`
+`deleteSchedule n/Joe d/tuesday-2` \
```
Comment (Gabau) [inline docs/UserGuide.md:235]: Maybe the shift number should be seperate from the date to make clearer parsing?

**Review by Gabau — CHANGES_REQUESTED** [review]
Some issues with command format and inconsistencies

File: docs/UserGuide.md (hunk 2 above)
Comment (Gabau) [reply docs/UserGuide.md:283]: Would keeping the square brackets be better? For the table view. Just noticed we haven't standardised.

Comment (fullfatwasabi) [reply docs/UserGuide.md:283]: hmm square brackets are for fields that are meant to be optional

File: docs/UserGuide.md (hunk 3 above)
Comment (fullfatwasabi) [reply docs/UserGuide.md:235]: Hmm seperate as in d/tueday 2 or d/tuesday shift/2? The current implementation in my PR is parsed as string then split which is fine I think?

Comment (Gabau) [reply docs/UserGuide.md:240]: I think its fine on second thought

**Pull request merged**